docs/releases/v2023.2.rst

```diff
@@ -0,0 +1,272 @@
+Gluon 2023.2
+============
+
+Important notes
+---------------
+
+Upgrades to v2023.2 and later releases are only supported from releases v2022.1 and later.
+This is due to migrations that have been removed to simplify maintenance.
+
+
+Deprecation of Tunneldigger VPN
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tunneldigger is set to be removed from core in the next Gluon release. It is recommended
+to migrate to fastd or WireGuard. Alternatively, tunneldigger will be moved to the
+community-packages repository and can be installed from there.
+
+
+Site changes
+------------
+
+Image customization
+~~~~~~~~~~~~~~~~~~~
+
+`GLUON_FEATURES` as well as `GLUON_PACKAGES` have been replaced by a more flexible customization framework
+based on a Lua DSL. Package and feature selection can be specified more granularly at both target and device level.
+
+Additionally, this framework also allows communities to specify which devices should or should not be built.
+For more information, see the :ref:`documentation <site-image-customization>`.
+
+
+Added hardware support
+----------------------
+
+armsr-armv7
+~~~~~~~~~~~
+
+- ARM
+
+  - SystemReady 32-bit (EFI)
+
+
+armsr-armv8
+~~~~~~~~~~~
+
+- ARM
+
+  - SystemReady 64-bit (EFI)
+
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- AVM
+
+  - FRITZ!Repeater 1750E
+
+- Sophos
+
+  - AP100
+  - AP100c
+  - AP55
+  - AP55c
+
+- TP-Link
+
+  - Archer C60 (v1)
+  - EAP225-Outdoor v3
+  - TL-WR2543N/ND (v1)
+
+
+ath79-mikrotik
+~~~~~~~~~~~~~~
+
+- MikroTik
+
+  - wAPR-2nD (wAP R)
+
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+- ZTE
+
+  - MF289F
+
+
+mediatek-filogic
+~~~~~~~~~~~~~~~~
+
+- ASUS
+
+  - TUF-AX4200
+
+- Cudy
+
+  - WR3000 (v1)
+
+- GL.iNet
+
+  - GL-MT3000
+
+- NETGEAR
+
+  - WAX220
+
+- Ubiquiti
+
+  - Unifi 6 Plus
+
+- ZyXEL
+
+  - NWA50AX Pro
+
+
+mpc85xx-p1010
+~~~~~~~~~~~~~
+
+- Enterasys
+
+  - WS-AP3715i
+
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - EAP615-Wall
+
+- Wavlink
+
+  - WS-WN572HP3 4G
+
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+- ASUS
+
+  - RT-AX53U
+
+- ZyXEL
+
+  - WSM20
+
+
+Removed hardware support
+------------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - Archer C60 (v1)
+  - RE355
+  - RE450 (v1)
+
+- Ubiquiti
+
+  - NanoBeam 5AC 19 (XC)
+  - NanoBeam M5 (XW)
+  - NanoStation Loco M2/M5 (XW)
+  - NanoStation M2/M5 (XW)
```

Comment:
Needs an explanation (footnote?) for devices that are removed temporarily because of critical issues, but that we expect to be added again when the issues have been solved.

Reply:
It's in the Knwon Issues that they are missing and will be readded once the issue is resolved. Maybe we could only mention it there.

If nobody were to readd them we could still remove that known issue in a future release and make a "Removed hardware support" annoucement for them.

And if it's fixed we can remove the Known Issue.

Reply:
Yes, It's a known issue for this release. I think this should be enough.

Reply:
As removed hardware support is always a topic of controversy, I'd prefer to make this as obvious as possible.